gluecodium/src/main/java/com/here/gluecodium/validator/LimePropertiesValidator.kt

```diff
@@ -63,6 +75,36 @@ internal class LimePropertiesValidator(private val logger: LimeLogger) {
             logger.error(limeProperty, "property in an interface cannot be marked with @Cpp(Ref)")
             result = false
         }
+        if (needsDocumentationComment(limeProperty, referenceMap) && !validateComments(limeProperty)) {
+            if (werrorPropertiesDocs) {
+                result = false
+            }
+        }
+        return result
+    }
+
+    private fun validateComments(limeProperty: LimeProperty): Boolean {
+        val propertiesDocsErrorMessage = "property must be documented; check 'docs/lime_markdown.md'"
+        var result = true
+
+        if (limeProperty.comment.isEmpty()) {
+            logger.maybeError(limeProperty, "main comment is empty; $propertiesDocsErrorMessage")
+            result = false
+        }
+
+        if (limeProperty.getter.returnType.comment.isEmpty()) {
+            logger.maybeError(limeProperty, "return of getter is not documented; $propertiesDocsErrorMessage")
```

Comment:
The same

Reply:
The documentation has been adjusted to contain the information about accepted syntax and conditions when warning/error occurs. I adjusted it for properties, functions and lambdas.

Moreover, `LimePropertiesValidator` has been adjusted to print more verbose warnings and contain the full reference to the docs. The user may use the link and open Gluecodium's GitHub.